client/my-sites/plan-features-2023-grid/actions.tsx

```diff
@@ -168,12 +183,23 @@ const LoggedInPlansFeatureActionButton = ( {
 		);
 	}
 
-	if ( ! availableForPurchase && forceDisplayButton ) {
-		return (
-			<Button className={ classes } disabled={ true }>
-				{ buttonText }
-			</Button>
-		);
+	const is2023OnboardingPricingGrid = isEnabled( 'onboarding/2023-pricing-grid' );
+	if ( ! availableForPurchase ) {
+		if ( is2023OnboardingPricingGrid ) {
+			return (
+				<Plans2023Tooltip text={ translate( 'Please, contact support to downgrade your plan' ) }>
```

Comment:
```suggestion
				<Plans2023Tooltip text={ translate( 'Please contact support to downgrade your plan.' ) }>
```

Reply:
Minor text suggestion ^

Reply:
@lucasmendes-design I am going with this suggestion 👍